python/server/qgsserverresponse.sip

```diff
@@ -0,0 +1,77 @@
+/***************************************************************************
+                          qgsserverresponse.h
+
+  Define response class for services
+  -------------------
+  begin                : 2016-12-05
+  copyright            : (C) 2016 by David Marteau
+  email                : david dot marteau at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+/**
+ * \ingroup server
+ * QgsServerResponse
+ * Class defining response interface passed to services QgsService::executeRequest() method
+ *
+ *  Note:
```

Comment:
These notes don't relate to api usage, so should be pulled out of the doxygen tags and just put in normal c++ comments